### ISSUE TITLE
Nodes all mixed up under iPadOS within Editor UI

### ISSUE THREAD

**Issue by mediana** [n8n-io/n8n#1989]
**Describe the bug**
All nodes will appear mixed up after saving a workflow.
Node position will never be saved afterwards.

**To Reproduce**
1. Create a new workflow with nodes
2. Save workflow
3. Refresh Page/Workflow

**Expected behavior**
Nodes should be placed like user defined using Editor UI.

**Environment (please complete the following information):**
 - OS: iPadOS
 - n8n Version 0.129.0

**Comment by mutdmour** [comment]
Thanks for sharing the issue. I will take a look soon.

**Comment by mutdmour** [comment]
@mediana can you please share which browser you are using? Is it safari?

**Comment by mediana** [comment]
Indeed, this happens with Safari, but also tried in Chrome and Brave with same behavior.

**Comment by bitsapien** [comment]
I tried reproducing the issue, this is what I got on Safari, before save:
<img width="1527" alt="image" src="https://user-images.githubusercontent.com/3462960/126065116-1578cb24-7763-47bc-82dc-7a175b804644.png">

After save and refresh as mentioned,
<img width="1526" alt="image" src="https://user-images.githubusercontent.com/3462960/126065125-dff95192-7b77-4904-aff9-c54baccde6f5.png">

The editor seems to be doing some kind of cleaning up of the nodes and arrows post reload.

Could you post a screenshot of the "before and afters"  in your case @mediana ?

**Comment by mediana** [comment]
Sure, here is a « before and after » example

![IMG_0478](https://user-images.githubusercontent.com/1455721/126150610-4eb91e94-2ff8-4938-a21b-dce650597138.jpeg)

![55A81610-30B9-4365-8610-6FCE74317985](https://user-images.githubusercontent.com/1455721/126150681-193338b6-97e2-4eb7-9c29-5cb0c18e353a.jpeg)

**Comment by krynble** [comment]
A small contribution after some testing that might aid @mutdmour is that after testing I realized it seems like the iPad is not saving the position changes.

I interchangeably tested editing a single workflow both on my computer and iPad, and it seems that position changes are not being reflected correctly on the iPad.

**Comment by mutdmour** [comment]
Prepared fix here https://github.com/n8n-io/n8n/pull/2016..

**Comment by Joffcom** [comment]
Hey @mediana,

It looks like this one was fixed a while back in `0.175.0`, I am going to mark this one as closed for now and if you do run into this again let me know and we can open it again.

**Comment by ecxs** [comment]
I tried building some worklows the other day on my brand nee and up-to-date iPad Air, and I’m running into exactly the same issue. Both in Safari and Chrome.

**Comment by janober** [comment]
Opening again. If I remember correctly, did we fix the issues but we had to roll that fix back again, as it messed up something else which was much worst.

**Comment by paulmeller** [comment]
Any update on this?

**Comment by Joffcom** [comment]
We’re taking some time to review and tidy up our issue tracker. It looks like this issue hasn’t had any recent activity for the past 6 months. To keep our backlog focused on current priorities, we’re closing this issue for now.

That said, this doesn’t mean the issue is forgotten! If this is still relevant or you have more information to add, feel free to reopen it or create a new issue and we’d be happy to revisit it.

Thank you for understanding and for contributing to the project!
